modules/ROOT/pages/apicat-create-descriptor-file-manually.adoc

```diff
@@ -2,6 +2,8 @@
 
 include::reuse::partial$non-inclusive-banner.adoc[]
 
+WARNING: To create or update Descriptor files manually, your files need to be saved using the correct character encoding format (UTF-8). Encoding problems may occur when the encoding of the Descriptor file doesn't match the expected encoding of API Catalog CLI.
```

Comment:
To avoid passive voice, you can say:

WARNING: To create or update Descriptor files manually, you have to save your files using UTF-8 character encoding format. Encoding problems may occur when the encoding of the Descriptor file doesn't match the expected encoding of API Catalog CLI.